src/charm.py

```diff
@@ -202,7 +202,7 @@ def _process_env_file(self, event):
             if self.unit.is_leader():
                 self._state.env = env
         except ModelError as err:
-            logger.error(err)
+            logger.debug(f"env-file resource not found {err}")
```

Comment:
suggestion: By the way, I saw in another PR that the charm reviewers asked for lazy evaluation of strings. You might want to change this.